tests/backup/backup_helper.go

```diff
@@ -364,6 +389,18 @@ func CreateBackupByNamespacesWithoutCheck(backupName string, clusterName string,
 			Uid:  postRuleUid,
 		},
 	}
+
+	if os.Getenv("BACKUP_TYPE") == "Generic" {
```

Comment:
Dont you want to handle case-sensitive failures here.

Reply:
Done